tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/InstanceProviderTests.cs

```diff
@@ -66,6 +66,10 @@ public void KnownMetadataProvider_RespondsIfEnvironmentsAreKnown()
                 "login.partner.microsoftonline.cn", new[] { "login.windows.net", "login.microsoft.com", "login.partner.microsoftonline.cn" }, _logger);
             Assert.IsNotNull(result);
 
+            result = knownMetadataProvider.GetMetadata(
```

Comment:
Can we have a test around regional that confirms that when using PPE, an instance discovery call is not made ?

Reply:
You'll also need to fix the test `OtherCloudWithAuthorityValidationAsync`, as we still want to test with a "different cloud".

Reply:
> Can we have a test around regional that confirms that when using PPE, an instance discovery call is not made ?

Does this test already validate this?

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/5e29d349dde99775bf60f23812e2a2437bb0cca9/tests/Microsoft.Identity.Test.Unit/PublicApiTests/ClientCredentialWithRegionTests.cs#L398